### DISCUSSION TITLE
Vazoyu kim kırdı?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#15] [Zeka Sorusu]
Ahmet, Bayram, Cengizhan, Derviş ve Erol oyun oynarlarken, bunlardan biri yanlışlıkla antika bir
vazoyu kırıyor. Vazoyu kimin kırdığını hepsi biliyor. Sorguya çekildiklerinde aşağıdaki beyanatlarda
bulunuyorlar.
Ahmet: “Vazoyu Bayram kırdı.”
Bayram: “Vazoyu Derviş kırdı.”
Cengizhan: “Vazoyu ben kırmadım.”
Derviş: “Bayram yalan söylüyor.”
Erol: “Ahmet doğru söylüyor”

Soru 1: Beşi arasından doğru söyleyenlerin sayısının alabileceği kaç farklı değer vardır?

Soru 2: Beşi arasından en az iki tanesi doğru söylüyorsa hangisi vazoyu kırmış olamaz?

Soru 3: Beşi arasından en az iki tanesi yalan söylüyorsa hangisi vazoyu kırmış olamaz?

Soru 4: Vazoyu kırmamasına rağmen yalan söylüyor olabilecek kişiler hangileridir?

Soru 5: Kimin doğru söylediği, kimin yalan söylediği bilinmesine rağmen, hala vazoyu kıran kişi
belirlenemiyorsa , hangisi vazoyu kırmamasına rağmen yalan söylemiştir?

**Comment by slos8888** [comment]
s1:Doğru söyleyenlerin sayısı maksimum 3 olabilir, çünkü eğer 4 veya 5 kişi doğru söylüyorsa, çelişki oluşur. 3 kişi de aynı zamanda yalan söylemiş olur ve bu çatışmaya neden olur. Bu durumda beşi arasından doğru söyleyenlerin sayısının alabileceği farklı değer sayısı 1, 2 ve 3'tür.
s2:En az iki kişi doğru söylüyorsa, diğerleri yalan söylüyordur. Ahmet ve Erol'un doğru söylediğini biliyoruz. Bu nedenle Bayram, Cengizhan ve Derviş yalan söylüyorlar. Bu üç kişi arasından vazoyu kıran kişi olamaz.
s3:En az iki kişi yalan söylüyorsa, diğerleri doğru söylüyordur. Bayram ve Derviş'in yalan söylediğini biliyoruz. Bu nedenle Ahmet, Cengizhan ve Erol doğru söylüyorlar. Bu üç kişi arasından vazoyu kıran kişi olamaz.
s4:Vazoyu kırmayan ama yalan söylemiş olabilecek kişileri belirlemek için, sadece tek bir kişi yalan söylüyorsa, geri kalanların hepsi doğru söylüyor demektir. Bu nedenle Derviş yalan söylüyor olabilir.
s5:Eğer kimin doğru söylediği ve kimin yalan söylediği bilinmesine rağmen vazoyu kıran kişi belirlenemiyorsa, bu durumda sadece Cengizhan vazoyu kırmamış olabilir. Diğerleri antika vazoyu kırmış olabilirler.
